<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="NDI-LAP-368" name="Default test" tests="3" failures="0" timestamp="15 Mar 2017 19:12:10 GMT" time="0.05" errors="0">
  <testcase name="@BeforeClass BeforeClassMethod" time="0.0" classname="org.sel.quiz.tests.SignInTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at org.sel.quiz.utills.ApplicationClass.OpenWebApp(ApplicationClass.java:32)
	at org.sel.quiz.tests.SignInTest.BeforeClassMethod(SignInTest.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:175)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:107)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </failure>
  </testcase> <!-- @BeforeClass BeforeClassMethod -->
  <testcase name="@AfterClass AfterClass" time="0.0" classname="org.sel.quiz.tests.SignInTest">
    <skipped/>
  </testcase> <!-- @AfterClass AfterClass -->
  <testcase name="clickLogInButtonUsingAbsolutePath" time="0.003" classname="org.sel.quiz.tests.SignInTest">
    <skipped/>
  </testcase> <!-- clickLogInButtonUsingAbsolutePath -->
  <testcase name="OpenForgotPasswordInNewTab" time="0.002" classname="org.sel.quiz.tests.SignInTest">
    <skipped/>
  </testcase> <!-- OpenForgotPasswordInNewTab -->
  <testcase name="fetchTitle" time="0.003" classname="org.sel.quiz.tests.SignInTest">
    <skipped/>
  </testcase> <!-- fetchTitle -->
</testsuite> <!-- Default test -->
